### DISCUSSION TITLE
feat: add diagram component

### DISCUSSION THREAD

**Discussion by VR-Architect** [microsoft/fluentui-blazor#431] [Ideas]
# 🙋 Feature Request

Create a diagram component with drag and drop support. 

## 🤔 Expected Behavior

If there is a desire to move forward, I am happy to write full requirements document. This would be a complex component and I am willing to co-develop with someone who is more versed in the Fluent way of coding. 

## 💻 Examples
Ability to create, view, export, print diagrams like BPMN, ArchiMate, networking, enterprise architecture, business architecture, organization charts, and much more.

**Comment by vnbaaij** [comment]
Hi,

This is a to specialized sort of component for this library. We also lack the resources/time/knowledge to work on such an endeavor. Also, I do not see how much this would hook into Fluent that requires it to be part of the library.
I see possibility for you (and a group of enthusiasts) creating an extension library for this that could use some of the Fluent components as building blocks. Of course we would help with advice when possible.

I'm moving this to the discussion forum as it is not really an issue and to give others the opportunity to chime in.
